### DISCUSSION TITLE
Have the 1.2x releases been buggy for anyone else?

### DISCUSSION THREAD

**Discussion by anthonator** [cloudnative-pg/cloudnative-pg#3506] [Q&A]
We're happy users of CNPG but since 1.20 was released I've been having a hard time managing our clusters. CNPG was rock solid for us pre-1.2x. (We're on 1.21.1 now).

It seems like whenever an instance goes down the cluster gets in a weird state. For example, one of the instances in a 2 instance cluster went down and never came back up. We are seeing the following error when running `kubectl cnpg status`:

```
Communication issue detected: The operator was unable to receive the status from all the ready instances. This may be due to network restrictions such as NetworkPolicy and/or any other network plugin setting. Please verify your network configuration.
```

We don't have any `NetworkPolicy` resources in the cluster. I've also checked out the troubleshooting guide and didn't see anything relevant to us. I'm also unable to really do anything with the cluster. I can't scale up or down. Any mutating `kubectl cnpg` command will report success but seems to be ignored. There aren't any relevant logs in the operator pod. I'm able to see the cluster nodes and pods from the operator pod. I'm not sure how to get the cluster unstuck at this point.

We've had this happen many times now and it always seems to start with an instance going down and then management through the operator stops working. We're seeing this after upgrading CNPG and on fresh installs.

**Comment by gbartolini** [comment]
Hi @anthonator. Obviously our answer is that this should not be the case. We have had no indication whatsoever in that direction. On the contrary, we have evidence that stability has improved a lot in the last releases. I am talking primarily about these important changes that we have introduced: https://cloudnative-pg.io/documentation/1.22/installation_upgrade/#upgrading-to-121-from-a-previous-minor-version

However, they don't seem to have anything with what you suggest here. Do you think you could use any of the office hours to go through a screen sharing session with some of us to understand the nature of the problem? Or maybe we can organize one internally.
